app/sources/curation_concerns/resource_statistics_source.rb

```diff
@@ -10,60 +10,47 @@ def initialize(search_builder: ::CurationConcerns::AdminController.new.search_bu
     end
 
     def open_concerns_count
-      results_count("#{Hydra.config.permissions.read.group}:public")
+      results_count(public_read_group)
     end
 
     def authenticated_concerns_count
-      results_count("#{Hydra.config.permissions.read.group}:registered")
+      results_count(registered_read_group)
     end
 
     def restricted_concerns_count
-      # TODO: Replace this with a query that that returns all documents that
-      #       either lack the `read_access_group_ssim` key, or have the key
-      #       without the values of `public` or `registered`
-      repository.search(solr_arguments)["response"]["numFound"] - (authenticated_concerns_count + open_concerns_count)
+      results_count([not_regisitered_read_group,not_public_read_group])
     end
 
     def expired_embargo_now_authenticated_concerns_count
-      results_count(["#{Hydra.config.permissions.read.group}:registered", "embargo_history_ssim:*"])
+      results_count([registered_read_group, embargo_history_query])
```

Comment:
These are all queries -- why have some methods that end in `_query` and some that don't? Could you `s/_query//g` ?